apps/fabric-website/src/pages/Controls/BottomSheetPage/docs/android/BottomSheetOverview.md

```diff
@@ -1,5 +1,9 @@
 `BottomSheet` and `BottomSheetDialog` are used to display a modal list of menu items. They slide up over the main app content as a result of a user triggered action. `BottomSheet` retains state on configuration change.
 
-| BottomSheet with single line items                                                                                                                                                                    | BottomSheet with double line items                                                                                                                                                                    |
-| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| <img src="https://static2.sharepointonline.com/files/fabric/fabric-website/images/controls/android/surfaces/bottom_sheet_single_line.png" alt="Example of a Drawer on Android" style="width: 50%;" /> | <img src="https://static2.sharepointonline.com/files/fabric/fabric-website/images/controls/android/surfaces/bottom_sheet_double_line.png" alt="Example of a Drawer on Android" style="width: 50%;" /> |
+| BottomSheet with single line items                                                                                                                                                                                     | BottomSheet with double line items                                                                                                                                                                                     |
+| ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| <img src="https://static2.sharepointonline.com/files/fabric/fabric-website/images/controls/android/surfaces/bottom_sheet_single_line.png" alt="Example of a single line BottomSheet on Android" style="width: 50%;" /> | <img src="https://static2.sharepointonline.com/files/fabric/fabric-website/images/controls/android/surfaces/bottom_sheet_double_line.png" alt="Example of a double line BottomSheet on Android" style="width: 50%;" /> |
```

Comment:
I think all of these widths should be bumped up to somewhere between 60-100% to match the other control pages with two columns for their images. I can't specify the exact number because we're all over the place - probably because at a certain point the images get scaled by the markdown table or something.

Reply:
Updated the widths to 75% which is the size of the images for the PersonaChipView.